examples/test_extract.py

```diff
@@ -1,32 +1,39 @@
-import krec
-import argparse
-from pathlib import Path
+"""Extract KRec data from an MKV file and optionally save it to a new file.
 
-
-"""
-# Just extract and view
+Usage:
+# Extract and view
 python examples/test_extract.py input.mkv -v
 
 # Extract and save
 python examples/test_extract.py input.mkv -o output.krec -v
 """
 
+import argparse
+import logging
+import sys
+from argparse import Namespace
+from pathlib import Path
+
+import colorlogging
+
+import krec
+
 
-def main(args):
-    print(f"Extracting from: {args.input_file}")
+def main(args: Namespace) -> int:
```

Comment:
btw i usually prefer to put the argparse logic into main, instead of passing args. it is a minor thing but it avoids polluting the global namespace